dotnet/samples/KernelSyntaxExamples/Example51_StepwisePlanner.cs

```diff
@@ -111,12 +111,7 @@ private static IKernel GetKernel(bool useChat = false)
 
         var kernel = builder
             .WithLogger(ConsoleLogger.Logger)
-            .Configure(c => c.SetDefaultHttpRetryConfig(new HttpRetryConfig
-            {
-                MaxRetryCount = 3,
-                UseExponentialBackoff = true,
-                MinRetryDelay = TimeSpan.FromSeconds(3),
-            }))
+            .Configure(c => c.SetHttpHandlerFactory(new PollyHttpHandlerFactory<PollyRetryThreeTimesWithBackoff>()))
```

Comment:
Want to highlight how we are kind of admitting that you need to have Polly to use things like StepwisePlanner, now. If it remains that way, I think we should make it easier somehow.

Reply:
I would not use the `Need` because it all depends on the resiliency of the current infrastructure used by customer. 

Stepwise planner would work assuming high 9 figures of LLM service resiliency. That said, regarding which resiliency implementation used, yes, `Polly` seems a quite good example to work with but this could be still just one of the potential recommendations we have and could point to a future `readme` files with good resiliency extensions built by the community.